.gitignore

```diff
@@ -40,6 +40,6 @@ Scripts/gcovr
 Scripts/jenkins.py
 Scripts/ocunit2junit
 __pycache__/
-awsconfiguration.json
-amplifyconfiguration.json
+*awsconfiguration.json
```

Comment:
Please confirm that this pattern also continues to match the un-prefixed names `amplifyconfiguration.json` and `awsconfiguration.json`

Reply:
thanks! confirmed that git doesn't pick these files up